cultcargo/pfb-clean.yml

```diff
@@ -0,0 +1,32 @@
+_include:
+  - genesis/cult-cargo-base.yml
+
+cabs:
+  _include: genesis/pfb-clean/latest/uncabbedcabs.yml
+
+  pfb_init:
+    image: ${vars.cult-cargo.registry}/pfb-clean:cc${vars.cult-cargo.version}
+
+  pfb_grid:
```

Comment:
Also, the naming is up to you ultimately, but may I suggest dash rather than underscore, i.e. ``pfb-grid`` etc.

There's a bit of an unspoken convention elsewhere in stimela and cult-cargo that things connected by underscores are a single logical "token", while dashes separate tokens (think e.g. for a step labelled "image-1", stimela knows to treat the "1" as a suffix -- not so for "image_1"). From this point of view, subtasks of pfb ought to use dashes.

Reply:
@landmanbester an alternative I forgot to mention is to have dots in the name, e.g. ``pfb.init``. Kind of emphasizes the hierarchical nature of the cab. Up to you, whatever aesthetics you prefer here.

Reply:
I think . used to not work, that's what I tried initially. Seems to be working now